### PR TITLE
Fix multiple fatal errors

### PR DESCRIPTION
**Pull request by tblacker7**
Edit screen resulted in whitescreen of death in multiple scenarios. This commit fixes these:
 - overlayColor was being incorrectly referenced in both Dashboard and Hero blocks
 - deprecated versions were using RichText insert syntax, rather than simple variables. This has been rectified.

### CODE REVIEW

File: assets/scss/style_variations.scss [hunk 1]
```diff
@@ -56,6 +57,7 @@
   .nhsuk-care-card__content{
     background-color: $color_nhsuk-pale-yellow;
     padding-top: 0;
+    border: 0;
```
Comment (marcinca) [inline assets/scss/style_variations.scss]: please change to `border: none` or `border-width: 0;`

File: src/00-dashboard/index.js [hunk 2]
```diff
@@ -303,20 +303,19 @@ registerBlockType("nhsblocks/dashpanel", {
...
-                    </a>
+                        <a href={attributes.panelLink} className="nhsuk-promo__link-wrapper">
+                            <div class="nhsuk-panel-with-label" style={{
+                            backgroundImage: `url(${attributes.backgroundImage})`,
+                                backgroundSize: 'cover',
+                                backgroundPosition: 'center',
+                                backgroundColor: `${attributes.overlayColor}`
+                        }}>
+                                <h3 class="nhsuk-panel-with-label__label">
+                                    {attributes.panelTitle}
```
Comment (marcinca) [inline src/00-dashboard/index.js]: should it be `${attributes. panelTitle}` ?

File: src/03-reveal/index.js [hunk 3]
```diff
@@ -164,14 +164,14 @@ registerBlockType("nhsblocks/reveal1", {
 				}
 			},
 			save: ( {attributes} ) =>
-				<details className="nhsuk-details newstyle">
-					<summary className="nhsuk-details__summary" role="button" aria-controls="details-content-" aria-expanded="true" >
+				<details className="nhsuk-details">
+					<summary className="nhsuk-details__summary" role="button" aria-controls="details-content-" aria-expanded="false" >
 						<span className="nhsuk-details__summary-text" >
-							<RichText placeholder={__("Reveal Title", "nhsblocks")} value={attributes.revealTitle} />
+							`${attributes.revealTitle}`
```
Comment (marcinca) [inline src/03-reveal/index.js:170]: should it be ${attributes. revealTitle} ?

**Review by marcinca — CHANGES_REQUESTED** [review]

**Review by marcinca — APPROVED** [review]

File: assets/scss/style_variations.scss [hunk 4]
```diff
@@ -30,6 +30,7 @@
 .nhsuk-care-card.nhsuk-care-card--type.is-style-warning-callout { /* Warning Callout */
```
Comment (cehwitham) [inline assets/scss/style_variations.scss:30]: This seem a very specific selector?

File: src/08-card/index.js [hunk 5]
```diff
@@ -185,7 +185,7 @@ registerBlockType("nhsblocks/card1", {
 				<span role="text">
 				<span className="nhsuk-u-visually-hidden">Non-urgent advice: </span>
```
Comment (cehwitham) [inline src/08-card/index.js:186]: Question className? Also, is it semantic to have a role="heading" inside a heading?

File: src/10-hero-block/index.js [hunk 6]
```diff
@@ -87,7 +87,7 @@ registerBlockType("nhsblocks/heroblock", {
             </div>
             </InspectorControls>,
             <section className={`${className}  nhsuk-hero nhsuk-hero--image nhsuk-hero--image-description`} style={{
```
Comment (cehwitham) [inline src/10-hero-block/index.js:89]: className?

**Review by cehwitham — COMMENTED** [review]
Few questions.

File: assets/scss/style_variations.scss (hunk 4 above)
Comment (tblacker7) [reply assets/scss/style_variations.scss:30]: yes. Intentionally so. We dont want a yellow border on the other ones, just the warning callout

**Pull request merged**